lfs/config.go

```diff
@@ -134,11 +141,10 @@ func (c *Configuration) BatchTransfer() bool {
 		}
 
 		// Any numeric value except 0 is considered true
-		if n, err := strconv.Atoi(v); err == nil && n != 0 {
-			return true
-		}
+		n, _ := strconv.Atoi(v)
+		return n != 0
```

Comment:
That's true. Probably not something that'll change before Go 2.0 though. Could change it to:

```go
n, err := strconv.Atoi(v)
return n != 0 && err == nil
```